quantization/nlp/bert/migraphx/README.md

```diff
@@ -0,0 +1,67 @@
+# BERT QDQ Quantization in ONNX for MIGraphX
+There are two main steps for the quantization:
+1. Calibration is done based on SQuAD dataset to get dynamic range of floating point tensors in the model
+2. Q/DQ nodes with dynamic range (scale and zero-point) are inserted to the model
+
+After Quantization is done, you can evaluate the QDQ model by running evaluation script we provide or your own script.
+
+The **e2e_migraphx_bert_example.py** is an end-to-end example for you to reference and run.
+
+## Requirements
+* Please build from latest ONNX Runtime source (see [here](https://onnxruntime.ai/docs/build/eps.html#migraphx)) for now.
+We plan to include TensorRT QDQ support later in ONNX Runtime 1.11 for [ORT Python GPU Package](https://pypi.org/project/onnxruntime-gpu/)
+* MIGraphX 2.8 and above
+* ROCm 5.7 and above (For calibration data)
+* Python 3+
+* numpy 
+* The onnx model used in the script is converted from Hugging Face BERT model. https://huggingface.co/transformers/serialization.html#converting-an-onnx-model-using-the-transformers-onnx-package
```

Comment:
Suggest to add exact command lines to export onnx model, and optimize it to fp16 or int8.